### ISSUE TITLE
Shorten descriptions

### ISSUE THREAD

**Issue by S3B4S** [webgems/webgems#177]
Some descriptions for certain resources are very long, it'd be great if someone could go through all of these and shorten them. There shouldn't be too much of them.

Some examples:
- PHP: (The only proper) PDO tutorial
- Fullstack: freeCodeCamp curriculum

@lostdesign Maybe we should put some restriction on the amount of words that can be used for a resource?

**Comment by lostdesign** [comment]
Yeah they should be shortened. Approx. 2 sentenses would be nice, not sure how many words max we should do. Would be nice to have equally long descriptions in general. Looks better UI wise.

**Comment by S3B4S** [comment]
True, on my screen somewhere around 6 lines is what I'd say should be the max, so, taking the example of [awwwards](https://webgems.io/design?card=awwwards) we get

> The awards for design, creativity and innovation on the Internet. Get your inspiration here or even submit your own website!

Which is 20 words, 124 characters.

> Flatuicolorpicker is a project digging the Flat Color Picker which gives you the perfect colors for flat designs

18 words, 112 characters.

> A Medium-based blog, curating articles on modern UX Design practices, conventions, and ideas.

13 words, 93 characters.

So, having done my very reputable and reliable piece of research, I'd say we restrict it around 130 characters?

**Comment by S3B4S** [comment]
But I wouldn't be surprised that on bigger screen we can easily have more characters. Maybe we should consider omitting some words and replacing it with `...` when a box gets too long vertically?

**Comment by lostdesign** [comment]
> But I wouldn't be surprised that on bigger screen we can easily have more characters. Maybe we should consider omitting some words and replacing it with ... when a box gets too long vertically?

Imho best solution yeah!
